devtools/prod-envs/qcarchive-worker-openff-ani.yaml

```diff
@@ -0,0 +1,22 @@
+name: qcarchive-worker-openff-ani
+channel_priority: flexible
+channels:
+  - conda-forge
+  - pytorch-nightly
+  - defaults
+dependencies:
+  - python =3.7
+  - qcfractal
+  - qcengine =0.15.0
+
+  # ML calculations
+  - pytorch
+
+  # procedures
+  - geometric
+
+  # compute backends
+  - dask-jobqueue
+
+  - pip:
+    - torchani
```

Comment:
Let's pin this version.

Reply:
this should be at least 2.1.1 to get the ANI2x model as well.